src/cts/src/Clock.cpp

```diff
@@ -95,7 +95,7 @@ Box<double> Clock::computeSinkRegionClustered(
   auto yMin = std::numeric_limits<float>::max();
   auto yMax = std::numeric_limits<float>::lowest();
 
-  for (const auto [sinkX, sinkY] : sinks) {
+  for (const auto& [sinkX, sinkY] : sinks) {
```

Comment:
These are just float so there is no copying

Reply:
this does generate a warning though on my builds

Reply:
Sinks is a vector of std::pair, the unpack of the floats inside the vector is doing a copy of the std::pair before assignment to the floats, that's why we need the reference. It is a reference to the std::pair, not the floats.

Reply:
My gcc is "gcc (Debian 12.2.0-10) 12.2.0"